sherpa-onnx/csrc/online-recognizer.h

```diff
@@ -95,6 +95,7 @@ struct OnlineRecognizerConfig {
                          const std::string &hotwords_file, float hotwords_score)
       : feat_config(feat_config),
         model_config(model_config),
+        lm_config(lm_config),
```

Comment:
Thanks to @kamirdin

See also
https://github.com/k2-fsa/sherpa-onnx/pull/149#issuecomment-1751629811